client.go

```diff
@@ -213,17 +213,16 @@ func (c *Client) Connect(ctx context.Context) error {
 		return err
 	}
 
-	s, err := c.CreateSessionWithContext(ctx, c.cfg.session)
+	s, err := c.CreateSession(ctx, c.cfg.session)
 	if err != nil {
-		c.CloseWithContext(ctx)
+		c.Close(ctx)
 		stats.RecordError(err)
 
 		return err
 	}
 
-	if err := c.ActivateSessionWithContext(ctx, s); err != nil {
-		c.closeSession(ctx, s) // ignore error since we cannot handle it anyway
```

Comment:
Is this internal method dropped or accidentally incorporated in some other code place?

Reply:
I guess a leftover. Will fix

Reply:
Eh, you looked at the wrong side :) 

<img width="1160" alt="image" src="https://github.com/gopcua/opcua/assets/229005/6b864222-5140-4767-a850-3227a0953a34">